src/Orleans/AssemblyLoader/AssemblyLoader.cs

```diff
@@ -291,15 +81,15 @@ private static Assembly MatchWithLoadedAssembly(AssemblyName searchFor, IEnumera
                 var candidateFullName = assembly.FullName;
                 if (String.Equals(candidateFullName, searchForFullName, StringComparison.OrdinalIgnoreCase))
                 {
-                    return assembly;                    
+                    return assembly;
                 }
             }
             return null;
         }
 
         private static Assembly MatchWithLoadedAssembly(AssemblyName searchFor, AppDomain appDomain)
         {
-            return 
+            return
```

Comment:
@xiazen I wonder does that mean we support both as per #2145. There was a bit of discussion if the `AppDomain` be shimmed ("polyfilled") at https://gitter.im/dotnet/orleans?at=57b6079287f779f0690a9fca &ndash; also some NETStandard code on assembly loading (that likely doesn't work correctly).